tools/autograd/derivatives.yaml

```diff
@@ -67,6 +71,11 @@
 #     Note that the number of differentibale outputs can be modified by the
 #     'output_differentiability' entry (see above).
 #
+#     Across a differentiable function's derivatives set, it is not
+#     permitted to mix the use of "grad", "grads[INDEX]", and
+#     "grad_NAME". You must be consistent for that differentiable
```

Comment:
nit: "grad_{name}" ? To be consistent with the other sentence above.

Reply:
done